### ISSUE TITLE
Dataset problem

### ISSUE THREAD

**Issue by tanmayagarwal1803** [hessamb/lcnn#5]
Can you guide me with regards to TinyImagenet Training.We are not able to run it due to the code requiring all the images of imagenet12

**Comment by johndpope** [comment]
i recreate the algorithm in pytorch + celebrity image dataset - 
training is still running - but the loss is converving.
https://github.com/johndpope/LCNN-pytorch/tree/main/take2
